tools/lsp/ui/api.slint

```diff
@@ -466,15 +466,15 @@ export global Api {
 
     // ## Property Editor
     pure callback test-code-binding(element-url: string, element-version: int, element-offset: int, property-name: string, property-value: string) -> bool;
-    callback set-code-binding(element-url: string, element-version: int, element-offset: int, property-name: string, property-value: string);
-    callback set-color-binding(element-url: string, element-version: int, element-offset: int, property-name: string, property-value: color);
+    pure callback set-code-binding(element-url: string, element-version: int, element-offset: int, property-name: string, property-value: string);
```

Comment:
I haven't read the details, but callback that starts with set- are most likely not pure. Why are you marking them as pure?